src/null-safety/index.md

```diff
@@ -314,9 +60,29 @@ To find the most recent releases, see the
 Dart SDK archive, or the **Beta channel** section of the
 [Flutter SDK archive.][flutter-sdks]
 
+### Migrating an existing package or app {#migrate}
+
+For instructions on how to migrate your code to null safety,
+see the [migration guide][].
+
+
+### Creating a null-safe package or app {#create}
 
-Set the [SDK constraints](/tools/pub/pubspec#sdk-constraints)
-to require a [language version][] that has null safety support.
+The templates used by the [`dart create` command][`dart create`] and IDEs
```

Comment:
nit: i find this a bit hard to read with the link being both on the `dart create` and the word 'command'. I think it would be easier to read if the link was just on `dart create`

Reply:
Sure. I tend to think that links on code font are hard to see, but I do agree that it looks weird. I'll change it.